scripts/qa_crowbarsetup.sh

```diff
@@ -450,11 +450,11 @@ function iscloudver()
         local milestone=${v#*M}
         v=${v%M*}
         if [[ "$ver" -eq "$v" ]] && [[ $cloudsource =~ ^M[0-9]+$ ]] ; then
-            [ "${cloudsource#*M}" $operator "$milestone" ]
+            test "${cloudsource#*M}" $operator "$milestone"
```

Comment:
@jdsn and @aspiers, I changed the tests back. Maybe I'll take a look at shellcheck. I think it's written in Haskell, which should be fun :smile: